cypress/e2e/room-access-settings/room-access-settings.spec.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+
+import RoomUtils from "../utils/room-utils";
+import Chainable = Cypress.Chainable;
+
+
+
+describe("Check room access settings", () => {
+    const homeserverUrl = Cypress.env('E2E_TEST_USER_HOMESERVER_URL');
+    const email = Cypress.env('E2E_TEST_USER_EMAIL');
+    const password = Cypress.env('E2E_TEST_USER_PASSWORD');
+    const homeserverShortname = Cypress.env('E2E_TEST_USER_HOMESERVER_SHORT');
+    const today = new Date().toISOString().slice(0, 10).replace(/-/g, '');
+
+    beforeEach(() => {
+        cy.loginUser(homeserverUrl, email, password);
+    });
+
+    afterEach(() => {
+        // todo logout, otherwise the login test is not reliable aby more.
+        // todo delete room, otherwise the test user will end up with a million identical rooms after a while.
```

Comment:
I have a PR to add this : https://github.com/tchapgouv/tchap-web-v4/pull/170
(apprrrouuuuuuuuuuuuuuve ma PRRRRRRRRR 🐍 😵‍💫)